.github/SUPPORT.md

```diff
@@ -81,11 +54,7 @@ Blocks should be formatted with three backticks (\`\`\`)
 This should be used for any configuration snippets of Vector, Helm values, etc.
 and for Vector console log messages.
 
-If you are not aware of how to format text this way, it is explained in the
-[Markdown documentation on fenced code
-blocks](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).
```

Comment:
I think we probably should keep the link to documentation on how to block format, it can be rephrased to be like the "single word or phrase" section maybe.